torchsynth/util.py

```diff
@@ -52,6 +52,29 @@ def normalize(signal: Signal) -> Signal:
     return signal / max_sample
 
 
+def batch_resize(x: torch.Tensor, size: int):
```

Comment:
So are we really sure we need this? Can we just insist that batch size is divisible by DEFAULT_BATCH_SIZE and be done

Reply:
like if we just assert batch_size % DEFAULT_BATCH_SIZE = 0 doesn't that make our lives easier?